source/materials/MaterialProperties.cc

```diff
@@ -708,6 +715,8 @@ namespace opticalprops {
     LXe_mpt->AddConstProperty("SCINTILLATIONYIELD1", .03);
     LXe_mpt->AddConstProperty("SCINTILLATIONYIELD2", .97);
     LXe_mpt->AddConstProperty("ATTACHMENT", 1000.*ms, 1);
+    LXe_mpt->AddConstProperty("IONIZATIONENERGY",    21.9 * eV, 1);
+    LXe_mpt->AddConstProperty("FANOFACTOR",               0.29, 1);
```

Comment:
I've changed the ionization energy. However, I'm puzzled about the Fano factor ( in NEXT papers we used the value 20 sometimes...) : maybe we should review also the value we use in xenon, then.

Reply:
See, for instance, https://journals-aps-org.ezproxy.cern.ch/prb/pdf/10.1103/PhysRevB.68.054201.

Reply:
See, for instance 
[PhysRevB.68.054201.pdf](https://github.com/user-attachments/files/17001738/PhysRevB.68.054201.pdf), where they talk about discrepancies between calculations and measurements.

Reply:
> in NEXT papers we used the value 20 sometimes...

In gas? The discrepancy seems to be only in liquid. For gas I thought we always used 0.15, although the first paper above measured 0.29.

> [PhysRevB.68.054201.pdf](https://github.com/user-attachments/files/17001738/PhysRevB.68.054201.pdf), where they talk about discrepancies between calculations and measurements.

Not sure what to make of this... Should we use F=20 then?

Reply:
> > in NEXT papers we used the value 20 sometimes...
> 
> In gas? The discrepancy seems to be only in liquid. For gas I thought we always used 0.15, although the first paper above measured 0.29.

I was referring to liquid. Yes, we used 0.15, but it's probably time to review it, also.
> 
> > [PhysRevB.68.054201.pdf](https://github.com/user-attachments/files/17001738/PhysRevB.68.054201.pdf), where they talk about discrepancies between calculations and measurements.
> 
> Not sure what to make of this... Should we use F=20 then?

Reply:
I believe simulations in LXe need to go in a different direction because we need to introduce the anticorrelation between ionization and scintillation. This requires adapting nexus to this case, which may not be trivial. Also, we might want to use NEST (by the way, they use F=0.03 for LXe).